### ISSUE TITLE
Aksjemarked

### ISSUE THREAD

**Issue by LarsFredrikHolstTry** [LarsFredrikHolstTry/bawser#28]
Jeg mener personlig at aksjemarked er en veldig gøy funksjon, spesielt for brukere med penger å tape.
Det har vært en storsuksess på Mafioso og satser på at Bawser brukerne også vil like det.

**Comment by Cobrazking** [comment]
Helt enig, dette trenger vi. Men et muligens dumt spørsmål. Klarer du på noen som helst måte å koble opp mot den ekte børsen? Si f.eks 4-5 aksjer som følger virkelighetens priser. Men for å gjøre det brukbart ingame må vi sikkert gjøre det mulig å gire invisteringen (eller legge til x10 på fortjenesten pr dag).

**Comment by LarsFredrikHolstTry** [comment]
> Helt enig, dette trenger vi. Men et muligens dumt spørsmål. Klarer du på noen som helst måte å koble opp mot den ekte børsen? Si f.eks 4-5 aksjer som følger virkelighetens priser. Men for å gjøre det brukbart ingame må vi sikkert gjøre det mulig å gire invisteringen (eller legge til x10 på fortjenesten pr dag).

Yeah, vi må bare finne et gratis API med rimelig antall requests så går det, men da blir jo også aksjemarkedet kun tilgjengelig fra 09:00 til 16:00

**Comment by Cobrazking** [comment]
Hadde jo vært kult og litt virkelighetsnært :)
